src/Service.php

```diff
@@ -60,7 +60,7 @@ public function load(SearchQueryInterface $query): LocationCollection
                 static::TYPE => $queryType->equals(Enums\AddressPart::CITY())
                     ? static::CITIES
                     : static::ADDRESS,
-                static::COMPONENTS => static::COUNTRY . ConfigInterface::UKRAINE,
+                static::COMPONENTS => static::COUNTRY . $this->config->getCountry(),
```

Comment:
config->getCountry() should be converted to lower case using mb_strtolower because of missing validation in configuration classes